lua/telescope/mappings.lua

```diff
@@ -97,6 +97,7 @@ local telescope_map = function(prompt_bufnr, mode, key_bind, key_func, opts)
   opts = opts or {
     silent = true
   }
+  if opts.noremap == nil then opts.noremap = true end
```

Comment:
we should do this check for both:

```lua
opts = opts or {}
if opts.silent == nil then opts.silent = true end
if opts.noremap == nil then opts.noremap = true end
```

Something like that.

Then we can merge, looks like you were right about that!

Reply:
Done, thanks.